securedrop/sass/source.sass

```diff
@@ -297,8 +297,24 @@ header
       padding-top: 0
       min-height: unset
 
-    p
+    &> section:not(#locales-menu):not(#flashed)
+
+      width: 100%
+      padding-top: 10px
+      min-height: unset
+
+      &:first-child
+        @include ltr
+          border-right: 0
+        @include rtl
+          border-left: 0
+
+      &#return-visit
+        padding-left: 50px
```

Comment:
This actually highlighted a lack of polish with right-to-left/left-to-right handling of the index page, but that should probably be handled in a separate ticket/PR.